storage/interface.go

```diff
@@ -180,6 +180,12 @@ type Appender interface {
 	ExemplarAppender
 }
 
+// OptionalGetRef is used by downstream projects (e.g. Cortex) to avoid maintaining a parallel set of references.
+type OptionalGetRef interface {
```

Comment:
```suggestion
type GetRef interface {
```
I think it's enough to call it out as optional in comment, WDYT?

Reply:
IDC

Reply:
I care 🙃

Reply:
I changed it.

Reply:
Thanks